update nodeadm uninstall documentation (#1074)
awsdocs/amazon-eks-user-guide: latest/ug/nodes/hybrid-nodes-nodeadm.adoc

`--skip`
|FALSE

> |Phases of uninstall to be skipped. It is not recommended to skip any of the phase unless it helps to fix an issue.

Reviewer: ```suggestion
|Phases of uninstall to be skipped. It is not recommended to skip any of the phases unless it helps to fix an issue.
```

Author: done

`--help`
|FALSE
|Displays help message with available flag, subcommand and positional value parameters.	

|`-f`,

`--force`
|FALSE
|Force delete additional directories that might contain leftovers from the node process.

*WARNING*

> - This will delete all contents in default Kubernetes and CNI directories (/var/lib/cni, /etc/cni/net.d, etc). Do not use this flag if you store your own data in these locations.

Reviewer: ```suggestion
This will delete all contents in default Kubernetes and CNI directories (/var/lib/cni, /etc/cni/net.d, etc). Do not use this flag if you store your own data in these locations.
```

Author: Done

> - `Safe Handling` - The `./nodeadm uninstall --skip node-validation,pod-validation --force` command no longer deletes the /var/lib/kubelet directory started from nodeadm v1.0.9. This is because it may contain PODs volume and volume-subpath directories that sometimes include mounted node filesystem. Deleting mounted paths can lead to accidental deletion of the actual mounted node filesystem. Before manually deleting /var/lib/kubelet directories, carefully inspect all active mounts and unmount volume safely to avoid data loss.

Reviewer: ```suggestion
Starting from nodeadm `v1.0.9`, the `./nodeadm uninstall --skip node-validation,pod-validation --force` command no longer deletes the /var/lib/kubelet directory. This is because it may contain Pod volumes and volume-subpath directories that sometimes include the mounted node filesystem. 

*Safe handling tips* - Deleting mounted paths can lead to accidental deletion of the actual mounted node filesystem. Before manually deleting the /var/lib/kubelet directory, carefully inspect all active mounts and unmount volumes safely to avoid data loss.
```

Author: Done

`--help`
|FALSE
|Displays help message with available flag, subcommand and positional value parameters.	

|`-f`,

`--force`
|FALSE

> |Force delete additional directories that might contain leftovers from the node process.

Reviewer 2: Unless "leftovers" a commonly used term, let's update to something more descriptive.

Author: Ack. Updated to "Force delete additional directories that might contain remaining files from Kubernetes and CNI components." 

Reference logs:-
```
root@hybrid-node:/home/eksahybrid# ./nodeadm uninstall --skip node-validation,pod-validation --force
{"level":"info","ts":"2025-07-29T02:42:51.330Z","caller":"uninstall/uninstall.go:74","msg":"Loading installed components"}
{"level":"info","ts":"2025-07-29T02:42:51.330Z","caller":"uninstall/uninstall.go:83","msg":"Creating daemon manager.."}
{"level":"info","ts":"2025-07-29T02:42:51.331Z","caller":"uninstall/uninstall.go:114","msg":"Creating package manager..."}
{"level":"info","ts":"2025-07-29T02:42:51.331Z","caller":"uninstall/uninstall.go:116","msg":"Configuring package manager with","containerd source":"none"}
{"level":"info","ts":"2025-07-29T02:42:51.331Z","caller":"flows/uninstall.go:61","msg":"Uninstalling kubelet..."}
{"level":"info","ts":"2025-07-29T02:42:51.344Z","caller":"flows/uninstall.go:70","msg":"Stopping SSM daemon..."}
{"level":"info","ts":"2025-07-29T02:42:51.352Z","caller":"ssm/install.go:168","msg":"Uninstalling SSM agent..."}
{"level":"info","ts":"2025-07-29T02:43:00.004Z","caller":"flows/uninstall.go:126","msg":"Uninstalling kubectl..."}
{"level":"info","ts":"2025-07-29T02:43:00.009Z","caller":"flows/uninstall.go:132","msg":"Uninstalling cni-plugins..."}
{"level":"info","ts":"2025-07-29T02:43:00.027Z","caller":"flows/uninstall.go:138","msg":"Uninstalling IAM authenticator..."}
{"level":"info","ts":"2025-07-29T02:43:00.033Z","caller":"flows/uninstall.go:150","msg":"Uninstalling image credential provider..."}
{"level":"info","ts":"2025-07-29T02:43:00.035Z","caller":"flows/uninstall.go:54","msg":"Finished uninstallation tasks..."}
{"level":"info","ts":"2025-07-29T02:43:00.035Z","caller":"uninstall/uninstall.go:135","msg":"Force mode enabled, cleaning up additional directories..."}.     //// FORCE command additional files and directory 
{"level":"info","ts":"2025-07-29T02:43:00.035Z","caller":"cleanup/cleanup.go:58","msg":"Removing directory","path":"/var/lib/cni"}
{"level":"info","ts":"2025-07-29T02:43:00.035Z","caller":"cleanup/cleanup.go:58","msg":"Removing directory","path":"/etc/cni/net.d"}
{"level":"fatal","ts":"2025-07-29T02:43:00.036Z","caller":"nodeadm/main.go:55","msg":"Command failed","error":"cleaning up additional directories: removing directory /etc/cni/net.d: unlinkat /etc/cni/net.d: directory not empty"}
```
